### PR TITLE
Fix 42: Make work pages prettier

### LINKED ISSUE #42: Improve work example page design
Feat. "Use all the data that we have in the backend" and "make an obvious link to each project"

### PR DESCRIPTION
**Pull request by nmorduch**
Closes #42 but I obviously forgot to put that in the commit as usual.

### CODE REVIEW

**Comment by nmorduch** [comment]
Should incorporate #38

**Comment by nmorduch** [comment]
Incorporated #38 :)

File: core/models.py [hunk 1]
```diff
@@ -70,6 +70,9 @@ class WorkPage(Page):
                             help_text="Approximate date of project completion.")
     testimonial = RichTextField(blank=True)
     testimonial_credit = models.CharField(max_length=255, blank=True)
+    link = models.CharField(max_length=255, null=True, blank=True,
```
Comment (harrislapiroff) [inline core/models.py]: Should be a [`URLField`](https://docs.djangoproject.com/en/1.9/ref/models/fields/#urlfield), probably.

Comment (nmorduch) [inline core/models.py]: Sounds like a good idea.

Comment (nmorduch) [inline core/models.py]: All I have to do for this is change the name, right?

Comment (harrislapiroff) [inline core/models.py]: And make a migration. I think that should otherwise do it.

Comment (nmorduch) [inline core/models.py]: Do I have to add any imports?

**Comment by nmorduch** [comment]
Note that the way testimonials look is somewhat irrelevant because we're going to switch to how they are on the other branch anyway.

**Comment by harrislapiroff** [comment]
Noted.

File: core/models.py (hunk 1 above)
Comment (harrislapiroff) [inline core/models.py]: Should work as is. It's in Django `models` module, which is already imported.

**Comment by harrislapiroff** [comment]
With the URLField fix made, I'm happy to have this merged by anyone.

**Pull request merged**